### DISCUSSION TITLE
Distance and Time data

### DISCUSSION THREAD

**Discussion by teguhraharjo17** [eusonlito/GPS-Tracker#88] [General]
![image](https://github.com/user-attachments/assets/0849757a-4730-4e5e-9e78-21a044294850)

where can I get distance and time data like this speed data in the position table database? because I want to get the data in detail like this speed data.

**Comment by eusonlito** [comment]
The distance and the time in between are not stored in the database.

The time is the difference between the date_utc_at values for each position, and the distance is calculated either with an SQL query or in some places within the code.

GPS devices often include a buffer to store records internally when they are out of range, so the data sent to the server may not be sorted. Recalculating the distance and time for each position received can be costly.

**Comment by teguhraharjo17** [comment]
okee thank you !!!
